tests/connection/connection.cfg

```diff
@@ -1,4 +0,0 @@
-[imc]
```

Comment:
Ok.  That sounds like something each developer would need todo locally?  I can make a new patch that adds this file in, and then adds that command to the CONTRIBUTIONS.rst section that talks about running tests.  Sound good?